src/doc/rustdoc/src/write-documentation/the-doc-attribute.md

```diff
@@ -141,13 +141,22 @@ But if you include this:
 
 it will not.
 
+## At the module level
```

Comment:
I don't see how this change impact things. Could you precise that `doc(test(attr(...)))` is only appending to the parent module's `doc(test(attr(...)))` please?

Reply:
Sure, done.